### PR TITLE
[139] Product Page using HTML

### PR DESCRIPTION
**Pull request by olakanmigoodnessb**
Capture 1
Capture 2
Capture 3
Capture 4

### CODE REVIEW

File: submissions/goodnessOlakanmi/HTML/Product-page.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			<li><a href="#">Contact</a></li>
+		</ul>
+	</nav>
+
+<!--This section is for About the company--> 
+	<section class="sec-about">
+		<div class="row-black">
+			<div class="row">
+				<h1>We make your feet happy!</h1>
+
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product-page.html:28]: Remove whitespace

File: submissions/goodnessOlakanmi/HTML/Product-page.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+				<article>
+					Omman is a Nigeria based footwear and leatherworks establishment that<br> aims to makes artistic, beautiful, comfortable and designer footwears<br> for male and female, young and old. The aim is to make your feet<br> happy, and we will do just that.
+				</article>
+
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product-page.html:32]: Remove whitespace

File: submissions/goodnessOlakanmi/HTML/Product-page.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+			</div>
+
+			<div class="col-3">
+				<img src="images/samplee.jpg" width="300px" height="350px" alt="leopard-print">
+				<h4>Leopard Print slippers: <strong> &#x20A6; 3,000</strong></h4>
+          <p> These sllipers are the definition of casual<br> yet classy. They will pair perfectly with<br> your  casual friday wears. 
+          </p>
+			</div>
+		</div>
+	</section>
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product-page.html:57]: Format properly

File: submissions/goodnessOlakanmi/HTML/Product-page.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+<!--This sector is for Other similar products-->
+	<section>
+		<h2>Shop for similar products</h2>
+	<ul class="other-products">
+		<li><a href="#">Zebra Print Slippers</li>
+		<li><a href="#">Gold Slippers</li>
+		<li><a href="#">Fur Slides</li>
+	</ul>
+	</section>
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product-page.html:67]: Format properly

File: submissions/goodnessOlakanmi/HTML/Product-page.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+	<footer>
+		<ul class="soc-media">
+			<li><a href="#" class="fa fa-https://www.twitter.com">Twitter</a></li>
+			<li><a href="#" class="fa fa-https://www.facebook.com">Facebook</a></li>
+			<li><a href="#" class="fa fa-https://www.pinterest.com">Pinterest</a></li>
+		</ul>
+
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product-page.html:75]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/goodnessOlakanmi/HTML/Product-page.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+<div>
+	<section>
+		<head>
+			<meta charset="utf-8">
+			<title>Omman Calzature</title>
+		</head>
+</div>
+	</section>
```
[the remark this answers is not in the document]
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product-page.html]: I have removed the div and section.

File: submissions/goodnessOlakanmi/HTML/Product-page.html (hunk 1 above)
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product-page.html]: I have removed it.

File: submissions/goodnessOlakanmi/HTML/Product-page.html (hunk 2 above)
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product-page.html]: I have removed it.

File: submissions/goodnessOlakanmi/HTML/Product-page.html (hunk 3 above)
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product-page.html]: I have.

File: submissions/goodnessOlakanmi/HTML/Product-page.html (hunk 4 above)
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product-page.html]: I have.

File: submissions/goodnessOlakanmi/HTML/Product-page.html (hunk 5 above)
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product-page.html]: I have.

File: submissions/goodnessOlakanmi/HTML/Product-page.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+		<div class="row-black">
+			<div class="row">
+				<h1>We make your feet happy!</h1>
+				<article>
+					Omman is a Nigeria based footwear and leatherworks establishment that<br> aims to makes artistic, beautiful, comfortable and designer footwears<br> for male and female, young and old. The aim is to make your feet<br> happy, and we will do just that.
+				</article>
+					<button type="button" value="Read More" role="button">Read More</button>
+			</div>
+		</div>
+<!-- This section is for products-->
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product-page.html:29]: You did not close this section

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/goodnessOlakanmi/HTML/Product-page.html (hunk 7 above)
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product-page.html]: I have

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**